src/index.js

```diff
@@ -1,21 +1,62 @@
-import _ from 'lodash';
 import './style.css';
-import Icon from './icon.png';
-
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
-  
-    // Add the image to our existing div.
-    const myIcon = new Image();
-    myIcon.src = Icon;
-  
-    element.appendChild(myIcon);
-  
-    return element;
+// import _ from 'lodash';
+
+// let cloud = 'cloud';
+const array = JSON.parse(localStorage.getItem('Data')) || [];
+const listInput = document.querySelector('#listInput');
+let counter = 1;
+const list = document.querySelector('#list');
```

Comment:
Good job on making all the requirments , only this counter should be initialized to 0  as here the first item gets 2 as index

Reply:
@souad988 Ohh my bad 😄 This mistake happened while fixing the eslint. I forgot to update in the method.
Thanks for your kind effort and direction. 😺